File: app/src/main/java/com/allybros/superego/activity/PlayGroundActivity.java
package com.allybros.superego.activity;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;

import com.allybros.superego.R;
import com.allybros.superego.widget.SegoProgressBar;

public class PlayGroundActivity extends AppCompatActivity {
    SegoProgressBar segoProgress;
    EditText etStartPercent;
    EditText etEndPercent;
    Button bt;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_play_ground);
        segoProgress = findViewById(R.id.segoProgress);
        bt = findViewById(R.id.bt);
        etEndPercent = findViewById(R.id.etEndPercent);
        etStartPercent = findViewById(R.id.etStartPercent);

        bt.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                String startPercent = etStartPercent.getText().toString();
                String endPercent = etEndPercent.getText().toString();
                int intStartPercent;
                int intEndPercent;
                try {
                    intStartPercent = Integer.parseInt(startPercent);
                    intEndPercent = Integer.parseInt(endPercent);
                } catch (Exception e){
                    intStartPercent = 1;
                    intEndPercent = 99;
                }
                segoProgress.setNewPercent(intStartPercent, intEndPercent);
            }
        });

    }
}

File: app/src/main/java/com/allybros/superego/activity/playground/PlayGroundActivity.kt
package com.allybros.superego.activity.playground

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.activity.viewModels
import androidx.databinding.DataBindingUtil
import com.allybros.superego.R
import com.allybros.superego.databinding.ActivityPlayGroundBinding
import java.lang.Exception

class PlayGroundActivity : AppCompatActivity() {
    private val viewModel: PlayGroundVM by viewModels()
    lateinit var binding: ActivityPlayGroundBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = DataBindingUtil.setContentView(this, R.layout.activity_play_ground)
        binding.bt.setOnClickListener {
            val startPercent = binding.etStartPercent.text.toString()
            val endPercent = binding.etEndPercent.text.toString()
            var intStartPercent: Int
            var intEndPercent: Int
            try {
                intStartPercent = startPercent.toInt()
                intEndPercent = endPercent.toInt()
            } catch (e: Exception) {
                intStartPercent = 1
                intEndPercent = 99
            }
            binding.segoProgress.setNewPercent(intStartPercent, intEndPercent)
        }
    }
}